handler.go

```diff
@@ -121,26 +121,47 @@ func (mux *SlackMux) slackHandlerWrapper(w http.ResponseWriter, r *http.Request)
 		resultChan <- false
 		return resultChan
 	}
+
 	commands := strings.Fields(text)
-	slackCmd, ok := mux.commandMap[commands[0]]
-	if !ok {
-		if mux.defaultHandler == nil {
-			fmt.Fprint(w, commands[0]+" is not a valid command.")
-			resultChan <- true
-			return resultChan
-		}
-		slackCmd = *mux.defaultHandler
+	commandName := getCommandName(commands)
+
+	if len(commands) > 0 && mux.isCommandInvalid(commandName) {
```

Comment:
I think it is better to do this check in the `isCommandInvalid` function. It provides a general idea about the validness of the command. To have the same functionality, I believe you should also check whether `defaultHandler` exists in `isCommandInvalid` function if `commands` is empty.

```suggestion
	if mux.isCommandInvalid(commands) {
```